app/models/holiday.rb

```diff
@@ -4,7 +4,7 @@ class Holiday < ApplicationRecord
 
   validates_presence_of :date
 
-  scope :future, -> { where("holidays.date >= ?", Time.current.to_date) }
```

Comment:
How hard would it be to add a spec to this which would show that it's failing on Oracle?

Reply:
I'm not sure how to do that (other than testing against Oracle).

Reply:
Oh, I forgot, I thought our CI was testing against oracle

Reply:
It does for nucore-nu, but not open